{
  "success": false,
  "error": "Traceback (most recent call last):\n  File \"/home/USCHunt/PycharmProjects/slither/venv/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 501, in _run_solc\n    ret: Dict = json.loads(stdout)\n  File \"/usr/lib/python3.8/json/__init__.py\", line 357, in loads\n    return _default_decoder.decode(s)\n  File \"/usr/lib/python3.8/json/decoder.py\", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n  File \"/usr/lib/python3.8/json/decoder.py\", line 355, in raw_decode\n    raise JSONDecodeError(\"Expecting value\", s, err.value) from None\njson.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/home/USCHunt/PycharmProjects/slither/slither/__main__.py\", line 741, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n  File \"/home/USCHunt/PycharmProjects/slither/slither/__main__.py\", line 72, in process_all\n    compilations = compile_all(target, **vars(args))\n  File \"/home/USCHunt/PycharmProjects/slither/venv/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 628, in compile_all\n    compilations.append(CryticCompile(target, **kwargs))\n  File \"/home/USCHunt/PycharmProjects/slither/venv/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 116, in __init__\n    self._compile(**kwargs)\n  File \"/home/USCHunt/PycharmProjects/slither/venv/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 539, in _compile\n    self._platform.compile(self, **kwargs)\n  File \"/home/USCHunt/PycharmProjects/slither/venv/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 149, in compile\n    targets_json = _get_targets_json(compilation_unit, self._target, **kwargs)\n  File \"/home/USCHunt/PycharmProjects/slither/venv/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 271, in _get_targets_json\n    return _run_solc(\n  File \"/home/USCHunt/PycharmProjects/slither/venv/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 505, in _run_solc\n    raise InvalidCompilation(f\"Invalid solc compilation {stderr}\")\ncrytic_compile.platform.exceptions.InvalidCompilation: Invalid solc compilation Error: Duplicate experimental feature name.\n    --> /home/USCHunt/Documents/ethereum/smart-contract-sanctuary/ethereum/contracts/mainnet/proxies_and_implementations//0.6.2/new/0c14e822e43796d955a30b6d974f62031da845e3_TadUniswapMiningProxy.sol/0c14e822e43796d955a30b6d974f62031da845e3_TadUniswapMiningProxy.sol:1294:1:\n     |\n1294 | pragma experimental ABIEncoderV2;\r\n     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Identifier already declared.\n    --> /home/USCHunt/Documents/ethereum/smart-contract-sanctuary/ethereum/contracts/mainnet/proxies_and_implementations//0.6.2/new/0c14e822e43796d955a30b6d974f62031da845e3_TadUniswapMiningProxy.sol/0c14e822e43796d955a30b6d974f62031da845e3_TadUniswapMiningProxy.sol:1300:1:\n     |\n1300 | contract TadUniswapMining is Ownable, Pausable, TadUniswapMiningStorage {\r\n     | ^ (Relevant source part starts here and spans across multiple lines).\nNote: The previous declaration is here:\n   --> /home/USCHunt/Documents/ethereum/smart-contract-sanctuary/ethereum/contracts/mainnet/proxies_and_implementations//0.6.2/new/0c14e822e43796d955a30b6d974f62031da845e3_TadUniswapMiningProxy.sol/0c14e822e43796d955a30b6d974f62031da845e3_TadUniswapMiningProxy.sol:961:1:\n    |\n961 | contract TadUniswapMining is Ownable, Pausable, TadUniswapMiningStorage {\n    | ^ (Relevant source part starts here and spans across multiple lines).\n\n\n",
  "results": {}
}